ruby/exercises/palindromo.rb

```diff
@@ -7,3 +7,10 @@
   # 'Madam'
 # Ejemplo invalido:
 # 'Hello'
+def palindromo(str)
+  str = str.downcase
```

Comment:
Cuando sea posible hay que usar una variable auxiliar para no modificar las variables recibidas en un metodo
```suggestion
  aux_str = str.downcase
```